hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -562,7 +562,7 @@ public final class ScmConfigKeys {
           1000L;
 
   public static final String OZONE_SCM_HA_RATIS_SNAPSHOT_GAP
-      = "raft.server.snapshot.creation.gap";
+      = "ozone.scm.ha.ratis.server.snapshot.creation.gap";
```

Comment:
@Xushaohong Thanks for review,
"raft.server.snapshot.creation.gap" -- This is configuration provided by ratis to trigger / take snapshot.
"ozone.scm.ha.ratis.server.snapshot.creation.gap" -- This configuration is provided at ozone mapping to ratis, which can be updated. And is applicable for both manual snapshot and raits automatic snapshot.

So adding manual is not required.

Reply:
Hi @sumitagrawl
Actually, I mean these two: ScmConfigKeys.**OZONE_SCM_HA_RATIS_SNAPSHOT_THRESHOLD** and **OZONE_SCM_HA_RATIS_SNAPSHOT_GAP**.
They are both SCM side conf, but their similar names could be confusing for those not too familiar.
It might be better to add some comments to tell the difference or rename them.